taskcluster/kinds/finetune-student/kind.yml

```diff
@@ -89,6 +89,7 @@ tasks:
                     $MOZ_FETCHES_DIR/corpus.aln.zst
                     None
                     None
+                    None
```

Comment:
suggestion (non-blocking): set this to `0` where we don't care what it is. Based on what I see in https://firefox-ci-tc.services.mozilla.com/tasks/a-vmSSwNRd61tKZOI3Bnxw/runs/0/logs/live/public/logs/live.log, it looks like it gets converted to that anyways - but this will guard against any future changes that, eg: cause opustrainer not to accept the string `None` as a valid seed.

Reply:
What you see in the config `seed: 0` is not OpusTrainer's random seed but Marian's one. And it's a good catch! We should change this one as well. In this case, I agree it would be easier to just make it required and pass any other number like 1111 or 0 if we don't care about it.

Reply:
Suggestion (optional): If we're confused about seeds it would be nice to be specific with something like `marian-seed`, although I see that the confusion is from the log from opustrainer itself, which we do not control. I'm unsure if you can pipe the output of another command into another logger so that it reports as `[opustrainer]` but that is beyond the scope of this PR.

Reply:
I think having it as just `seed` is fine, we are just supposed to use it in all the random operations involved in training. We log opus trainer config separately as artifact.